css/main.css

```diff
@@ -1,3 +1,555 @@
+  background-image: url('../img/bg.jpg');
+  background-size: cover;
+  background-position-y: 50%;
+}
+
+.footer-section {
+  background-color: var(--green-black);
+  color: var(--white);
+  display: grid;
+  place-items: center;
+}
+
+.footer-detail p:first-child,
+.footer-subscribe p:first-child {
+  letter-spacing: 0.5rem;
+  font-size: 1rem;
+  padding: 0.8rem 0;
+}
+.footer-detail p:not(first-child) {
+  font-size: 0.9rem;
+  padding: 0.3rem 0;
+}
+
+.footer-detail {
+  padding: 2rem 3.5rem;
+  display: grid;
+  place-items: center;
+  text-align: center;
+}
+.footer-detail a {
+  color: var(--white);
+}
+.footer-subscribe {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  display: grid;
+  place-items: center;
+}
+.footer-subscribe form {
+  padding: 0 1rem;
+}
+.footer-subscribe form input {
+  background-color: transparent;
+  border: 1px solid var(--white);
+  padding: 0.5rem;
+  width: 100%;
+  margin-top: 0.5rem;
+}
+.footer-subscribe form button {
+  background-color: var(--white);
+  color: var(--green-black);
+  margin-top: 0.5rem;
+  width: 100%;
+}
+.footer-icons * {
+  margin: 3rem 0.5rem;
+  cursor: pointer;
+}
+.footer-map iframe {
+  height: 37.5rem;
+  width: 100%;
+}
+.footer-crediting {
+  text-align: center;
+  color: var(--red-brown);
+  background-color: var(--light-grey);
+  padding: 3rem 0;
+}
+.main-help {
+  display: grid;
+  place-items: center;
+  text-align: center;
+  background-color: var(--white);
+  color: var(--red-brown);
+  margin: 5rem 0;
+  gap: 4rem;
+}
+.main-help-box * {
+  margin: 1rem 3rem;
+}
+@media screen and (min-width: 481px) {
+  .desktop-login,
+  .desktop-letschat {
+    display: inline;
+  }
+  .bottom-btn {
+    border-radius: 0%;
+    font-family: var(--fjalla-one);
+    padding: 0.9rem 2rem;
+  }
+  header {
+    position: sticky;
+    top: 0;
+    z-index: 999;
+  }
+  label[for='menu'],
+  #menu,
+  .mobile-nav {
+    display: none;
+  }
+  .desktop-nav {
+    display: grid;
+    grid-auto-flow: column;
+  }
+  .desktop-nav li {
+    padding-left: 1rem;
+  }
+  nav,
+  .nav-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  nav,
+  .header-section {
+    height: 3rem;
+  }
+  .main-hero {
+    padding-top: 3rem;
+    height: 40rem;
+  }
+  .main-hero .fjalla {
+    font-size: 7rem;
+    letter-spacing: 1rem;
+  }
+  .main-collection {
+    display: grid;
+    grid-auto-flow: column;
+  }
+  .main-about-shop {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-areas:
+      'ab ab p4 p5'
+      'p6 p6 bu bu';
+  }
+  .main-about {
+    grid-area: ab;
+    line-height: 1.5rem;
+    padding: 5rem;
+  }
+  .main-about * {
+    padding: 0.5rem 0;
+  }
+  .desktop-pic4 {
+    display: block;
+    grid-area: p4;
+    background-image: url('../img/desktop/4.webp');
+    background-size: cover;
+  }
+  .desktop-pic5 {
+    display: block;
+    grid-area: p5;
+    grid-column: 4/5;
```

Comment:
Typically, you will either use grid-area spec or assign to rows and columns. You shouldn't generally need to do both. Can you talk a little about why you chose to do this here?

Reply:
Sorry, personal habit, it has been removed now